testing-guidance/workflow-tests/animation/animation-editor-workflows.md

```diff
@@ -0,0 +1,120 @@
sues/10961) <ul><li>Add multiple Node Groups.</li><li>Have Node Groups are created on multiple levels of the graph.</li><li>Move nodes between groups.</li><li>Remove Nodes from Node Groups via Remove Node.</li><li>Rename the Node Groups via Double-Click and Right-Click → Rename. [o3de/o3de#13279](https://github.com/o3de/o3de/issues/13279) </li><li>Save and Close then Re-Open the Anim Graph and add nodes to an existing Node Group. [o3de/o3de#13426](https://github.com/o3de/o3de/issues/13426) </li></ul></li><li>Apply Motions and adjust properties to the motion nodes before wiring each output pose to the pose inputs of the Blend Two node.</li><li>Save the AnimGraph</li><li>Click the Play/Activate AnimGraph button to play the animation in the Animation Editor.</li><li>Modify the parameter values as the AnimGraph plays.</li></ol> | <ul><li>AnimationGraph can be created.</li><li>Nodes can be created via the Node Palette View or the Right-Click Palette which is searchable. [o3de/o3de#10963](https://github.com/o3de/o3de/issues/10963) </li><li>The nodes found in the Right-Click context menu have colored icons next to them. [o3de/o3de#11689](https://github.com/o3de/o3de/issues/11689) </li><li>Parameters can be created, modified, and applied to Transition Lines.</li><li>Parameter details are displayed in the Inspector . [o3de/o3de#10965](https://github.com/o3de/o3de/issues/10965) </li><li>When modifying the parameters verify they Inspector widget renders as expected. [o3de/o3de#13234](https://github.com/o3de/o3de/issues/13234) </li><li>Actions can be created, updated, and deleted from Nodes via the Inspector.</li><li>Source Nodes with sub-graphcs can be entered to define its behavior.</li><li>Node groups can be managed as expected: <ul><li>Multiple node groups can be created.</li><li>Nodes can be added to a node graph via the Right-Click → Assign to Node Group.</li><li>Nodes can be moved between node groups. [o3de/o3de#13416](https://github.com/o3de/o3de/issues/13416) </li><li>Nodes can only be added to node groups on the same level as the node. [o3de/o3de#13232](https://github.com/o3de/o3de/issues/13232) </li><li>Nodes can be removed from groups via the right-click context menu's remove node and unchecking the node from the group from the "Assign to Node Group" menu. [o3de/o3de#13470](https://github.com/o3de/o3de/pull/13470) </li><li>Node Groups can be managed from the right-click context menu.</li><li>Nodes can be added to an existing Node Group from a previously created AnimGraph. [o3de/o3de#13426](https://github.com/o3de/o3de/issues/13426) </li></ul></li><li>When the Play/Activate AnimGraph button is pressed the animation will play according to the current parameter value.</li><li>When the parameter values are adjusted from the Parameter view, the rendered animation will animate arccording to the conditions you've defined.</li><li>Anim Graphs can be saved and found on the file system as well as reopened and modified if closed from the editor.</li></ul> |
```

Comment:
suggestion: It's hard to leave comments on these sections but overall looks good, small thing is to remove the GHI link from the heading of the last row in the left column (this can be moved to the 'things to look for' column instead.

Reply:
Located on line 69 under the Modify Actor Joins workflow. Relocated the GHI from that section to the things to watch for entry.